tests/runtime/kubernetes/test_kubernetes_executor.py

```diff
@@ -142,9 +142,9 @@ def ensure_dag_expected_state(self, host, execution_date, dag_id,
         # Maybe check if we can retrieve the logs, but then we need to extend the API
 
     def start_dag(self, dag_id, host):
-        result = self.session.get(
+        result = self.session.post(
             'http://{host}/api/experimental/'
-            'dags/{dag_id}/paused/false'.format(host=host, dag_id=dag_id)
+            'dags/{dag_id}/pause/false'.format(host=host, dag_id=dag_id)
```

Comment:
Why are we changing endpoint from `paused` to `pause`, is that intended?

Reply:
@kaxil yes, because paused is a question, this is a directive.

Paused is "is this dag paused?"

Pause is "set this to paused"

Reply:
Let's change the name in a separate PR and keep this one limited to changing GET to POST please

Reply:
ok